t/18-backend-qemu.t

```diff
@@ -89,11 +89,11 @@ subtest 'using Open vSwitch D-Bus service' => sub {
 
 my $fake_qmp_answer;
 $backend_mock->redefine(handle_qmp_command => sub { push @{$called{handle_qmp_command}}, $_[1]; $fake_qmp_answer });
-$backend->power({action => 'off'});
+combined_like { $backend->power({action => 'off'}); } qr/[debug]/s, 'Debug message logged for power off';
```

Comment:
I think it would be good idea to check part of the debug message! not sure how can you be sure that it is the expected bebug message. I guess is the only one in the `power` but it make sense to check for something more specific. like `POWER.*off` or `expected_shutdown`. wdyt?

Reply:
You are right, this is the unhandled output:

```
3: [2025-05-13T15:48:59.157378Z] [debug] [pid:2653] POWER: action: off, expected_shutdown: 1
3: [2025-05-13T15:48:59.157950Z] [debug] [pid:2653] POWER: action: acpi, expected_shutdown: 1
3: [2025-05-13T15:49:01.013982Z] [debug] [pid:2653] POWER: action: off, expected_shutdown: 1
3: [15:49:02] t/18-backend-qemu.t ......................... ok     4538 ms ( 0.04 usr  0.00 sys +  3.72 cusr  0.81 csys =  4.57 CPU)
```

I'll add a proper string :)